### DISCUSSION TITLE
can you make a pkgbuild

### DISCUSSION THREAD

**Discussion by Kreijstal** [asctime/Ev2W#29] [Ideas]
to use with pacman of the msys2 project
alternatively show how to build this

**Comment by asctime** [comment]
While a pkgbuild is certainly technically possible from my end, the MSYS2 team and I operate by apparently very different philosophies from a release perspective: Most crucially MSYS2 is a psuedo-cygwin-centric Arch-Linux style rolling-release that this project probably couldn't remain compatible with.  There is also the issue of back-flipping their prior statements to me concerning their ongoing interest in GTK2..

For building the project I still use the MinGW (GCC 8.3.0) 2018 but 10,x is probably fine (I understand there were multiple issues with MinGW GCC 9 and it seemed pretty quickly bumped out, but I don't remember the details) Don't use any of the other variants like Clang, etc. Essentially, if your MinGW64 is using the GCC 10 stack and compiles their Pidgin pkgbuild, it should be able to compile Ev2W.

Give it a go and let me know.
